package.json

```diff
@@ -7,15 +7,18 @@
     "scssfiles": "src/*.scss src/**/*.scss"
   },
   "devDependencies": {
+    "@babel/core": "^7.1.0",
+    "@babel/plugin-proposal-async-generator-functions": "^7.1.0",
+	"@babel/plugin-transform-async-to-generator": "^7.1.0",
```

Comment:
Nit: a few lines in `package.json` seem to have a different indentation style. Let's align those with the rest (4 spaces I think?)